### ISSUE TITLE
Back To Top button

### ISSUE THREAD

**Issue by willgarrett64** [MK-Codecademy/Rest-Countries-API#58]
The list of countries can be quite long, and as you scroll down the filter/search bar disappear. It would be nice to have a way to quickly get back to the top

- Style a sort of button that says "back to top" or "scroll up", or something along those lines
- Clicking button scrolls the user back to the top of the page
- Button to appear only after user has scrolled past the search/filter bar
- Position the button somewhere that looks good on mobile and desktop

This could be it's own React component, but doesn't necessarily need to be.
